src/spyglass/spikesorting/analysis/v1/group.py

```diff
@@ -88,16 +88,19 @@ def filter_units(
         include_labels = np.unique(include_labels)
         exclude_labels = np.unique(exclude_labels)
 
+        if include_labels.size == 0 and exclude_labels.size == 0:
+            # if no labels are provided, include all units
+            return np.ones(len(labels), dtype=bool)
```

Comment:
Is this likely? The method getting invoked without labels feels like an err state to me, but maybe this is here to let unfiltered data get pushed though to the next stage

Reply:
It is just to let data to be passed through. There are some cases where it doesn't matter.